src/components/UserProfileSettings.tsx

```diff
@@ -29,7 +29,6 @@ export function UserProfileSettings({ user }: UserProfileSettingsProps) {
     const { error } = await logout()
     if (error) {
       logAndToastError(`Error logging out: ${error.message}`)
-      return
```

Comment:
we shouldn't stop the user from proceeding the the login page if the logout fails.